__app/component/WakeLock/README.md

```diff
@@ -0,0 +1,67 @@
+## WakeLock Component Documentation
+
+### 1. Happy Flow 
+
+```jsx
+<WakeLock />
```

Comment:
You develop as a service and included as a component @Alok30

Reply:
Updated .Please check